extensions/wikia/CreateNewWiki/classes/Starters.class.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Wikia\CreateNewWiki;
+
+use Wikia\SwiftStorage;
+
+/**
+ * A helper class for getting a proper starter wiki based on a given language
+ *
+ * @author macbre
+ */
+class Starters {
+
+	const STARTER_DUMPS_BUCKET = 'starter';
+
+	/**
+	 * starters map: langcode => database name
+	 *
+	 * "*" is default
+	 */
+	static private $mStarters = [
```

Comment:
As you already touched this piece maybe you want to join it with https://github.com/Wikia/config/blob/dev/CommonExtensions.php#L2332
so we don't have duplicated code.